client/packages/programs/src/JsonForms/common/components/Number.tsx

```diff
@@ -39,20 +57,11 @@ const UIComponent = (props: ControlProps) => {
         <FormLabel sx={{ fontWeight: 'bold' }}>{label}:</FormLabel>
       </Box>
       <Box flexBasis={FORM_INPUT_COLUMN_WIDTH}>
-        <NumericTextInput
-          type="number"
-          InputProps={{
-            sx: { '& .MuiInput-input': { textAlign: 'right' } },
-          }}
-          onChange={value => {
-            setLocalData(value);
-            onChange(value);
-          }}
-          disabled={!props.enabled}
-          error={error}
-          helperText={errors}
-          value={localData ?? ''}
-        />
+        {schema.minimum !== undefined && schema.minimum >= 0 ? (
+          <PositiveNumberInput {...inputProps} />
```

Comment:
This check might not be needed since the `PostitiveNumberInput` only allows for inputs where n >= 1 🤔

Reply:
Yes! I changed it in the adherence status control PR...

We don't even need the PostitiveNumberInput we could just set the minimum for the existing number control...

Reply:
> This check might not be needed since the `PostitiveNumberInput` only allows for inputs where n >= 1 🤔 ... props can directly be passed through PostitiveNumberInput if check is removed

But isn't this checking for _which_ component to display, not *whether* to display "PositiveNumberInput"? If `minimum` is a positive number then show the "PositiveNumberInput" component, but otherwise show "NumericTextInput" (i.e. allow negatives)?

Reply:
Oh, will the AST, ALT etc allow for negative inputs? 🤔